docs/reference/ccr/overview.asciidoc

```diff
@@ -29,6 +29,41 @@ Replication can be configured in two ways:
 
 NOTE: You must also <<ccr-requirements,configure the leader index>>.
 
+When you initiate replication either manually or through an auto-follow pattern, the
+follower index is created on the local cluster. Once the follower index is created,
+the <<remote-recovery, remote recovery>> process copies all of the Lucene segment
+files from the remote cluster to the local cluster.
+
+When initiating following manually using the API, by default the recovery process
+will be asynchronous in relationship to the
+{ref}/ccr-put-follow.html[create follower request]. The request will return prior
+to the <<remote-recovery, remote recovery>> process being completed. If you
+would like to wait on the process to complete, you can use the
+`wait_for_active_shards` parameter to do so.
```

Comment:
```suggestion
`wait_for_active_shards` parameter. For example:
```

Reply:
I'm not sure `For example` makes sense here. That code is commented out. It only sets up the test for the falling behind example. Wait_for_active_shards is defined and shown on the api page.

Reply:
You're right, nevermind!